docs/design/add-in-commands.md

```diff
@@ -1,38 +1,69 @@
+ms.date: 03/13/2023
 ms.localizationpriority: high
 ---
 
-# Add-in commands for Excel, PowerPoint, and Word
+# Add-in commands
 
 Add-in commands are UI elements that extend the Office UI and start actions in your add-in. You can use add-in commands to add a button on the ribbon or an item to a context menu. When users select an add-in command, they initiate actions such as running JavaScript code, or showing a page of the add-in in a task pane. Add-in commands help users find and use your add-in, which can help increase your add-in's adoption and reuse, and improve customer retention.
 
 > [!NOTE]
+>
 > - SharePoint catalogs do not support add-in commands. You can deploy add-in commands via [Integrated Apps](/microsoft-365/admin/manage/test-and-deploy-microsoft-365-apps) or [AppSource](/office/dev/store/submit-to-appsource-via-partner-center), or use [sideloading](../testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) to deploy your add-in command for testing.
 > - Content add-ins do not currently support add-in commands.
 
-> [!IMPORTANT]
-> Add-in commands are also supported in Outlook. For more information, see [Add-in commands for Outlook](../outlook/add-in-commands-for-outlook.md).
+## Types of add-in commands
+
+There are two types of add-in commands, based on the kind of action that the command triggers.
+
+- **Task pane commands**: The button or menu item opens the add-in's [task pane](task-pane-add-ins.md). You add this kind of add-in command with markup in the manifest. The "code behind" the command is provided by Office.
+  ![A button that opens a task pane on the Outlook ribbon.](../images/commands-task-pane-button-1.png)
+- **Function commands**: The button or menu item runs any arbitrary JavaScript. The code almost always calls APIs in the Office JavaScript Library, but it doesn't have to. This type of add-in typically displays no UI other than the button or menu item itself. Note the following about function commands:
+  - The function that is triggered can call the [displayDialogAsync](/javascript/api/office/office.ui?view=common-js&preserve-view=true#office-office-ui-displaydialogasync-member(1)) method to show a dialog, which is a good way to display an error, show progress, or prompt for input from the user. If the add-in is configured to use a [shared runtime](../testing/runtimes.md#shared-runtime), the function can also call the [showAsTaskpane](/javascript/api/office/office.addin#office-office-addin-showastaskpane-member(1)) method.
+  - The runtime in which the function command runs is a full [browser-based runtime](../testing/runtimes.md#browser-runtime). It can render HTML and call out to the Internet to send or get data.
+![A button that executes a function on the Outlook ribbon.](../images/commands-uiless-button-1.png)
 
-*Figure 1. Add-in with commands running in Excel Desktop*
+## How do add-in commands appear?
```

Comment:
I reworked that section to remove some images and flow better. I think it is easier to scan for your host, instead of risking people missing the information on another tab. Please let me know if you agree.

Reply:
That flows better. Thanks!